model/trace/messaging.yaml

```diff
@@ -164,3 +164,11 @@ groups:
         tag: tech-specific-gcp-pubsub
         requirement_level:
           conditionally_required: If the message type has an ordering key set.
+      - ref: messaging.gcp_pubsub.message.delivery_attempt
+        tag: tech-specific-gcp-pubsub
+        requirement_level: required
+      - ref: messaging.gcp_pubsub.ack_deadline_seconds
```

Comment:
is it a message/batch/destination/consumer property? Based on https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/pubsub_subscription is seems a destination property.

I.e. should we add another namespace like `messaging.gcp_pubsub.destination.ack_deadline_seconds` or `messaging.gcp_pubsub.subscription.ack_deadline_seconds`

Reply:
One argument for `subscription` - you'd might wat to add subscription name at some point or may be some other properties

Reply:
It is a subscription property, but also set per request for a message. I'm not sure what namespace would be better
subscription.ack_deadline_seconds or message.ack_deadline_seconds?

@hongalex

Reply:
Discussed with Alex, .message makes sense since this refers to the message level set deadline.